com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/bundle/HTML5Bundler.java

```diff
@@ -53,6 +54,10 @@ public class HTML5Bundler implements IBundler {
     private static final String SplitFileJson = "archive_files.json";
     private static int SplitFileSegmentSize = 2 * 1024 * 1024;
 
+    // previously it was hardcoded in dmloader.js
+    private long WasmSize = 2000000;
+    private long WasmjsSize = 250000;
+    private long AsmjsSize = 4000000;
```

Comment:
I suppose next step would be to actually use the size from the build instead of these hardcoded values.

Reply:
It's already implemented here https://github.com/defold/defold/pull/8682/files#diff-a160fbb18e998e49c493662cb7425be615a6c5a985efcbbb751d7ff03c421d39R97-R99